packages/ui/src/components/Switch/shared.ts

```diff
@@ -12,19 +12,36 @@ export interface CustomSwitchProps extends Omit<SwitchBaseProps, 'classes'> {
  */
 export type SwitchClasses = {
   root: string;
-  rootSizeL: string;
```

Comment:
There should be at least `rootSizeSM`, `rootSizeM`, `rootSizeL` and `rootSizeXL`